mkosi/__init__.py

```diff
@@ -2191,6 +2192,8 @@ def install_openmandriva(args: CommandLineArguments, root: str, do_run_build_scr
     if not do_run_build_script and args.bootable:
         packages |= {"kernel-release-server", "binutils", "systemd-boot", "dracut", "timezone", "systemd-cryptsetup"}
         configure_dracut(args, root)
+    if args.network_veth:
+        packages |= {"systemd-networkd"}
```

Comment:
Could you split this into a separate commit or at least update the commit message, since this is unrelated to the architectures?

Reply:
Well i forgot to mention in commit about this